book/src/user/shielded-scan.md

```diff
@@ -1,90 +1,75 @@
+ 
 For now, we only support Sapling, and only store transaction IDs in the scanner results database.
-Ongoing development is tracked in issue [#7728](https://github.com/ZcashFoundation/zebra/issues/7728).
+
+Ongoing development is tracked in issue [#7728](https://github.com/ZcashFoundation/zebra/issues/7728) and [#8573](https://github.com/ZcashFoundation/zebra/issues/8573).
 
 ## Important Security Warning
 
 Zebra's shielded scanning feature has known security issues. It is for experimental use only.
 
-Do not use regular or sensitive viewing keys with Zebra's experimental scanning
-feature. Do not use this feature on a shared machine. We suggest generating new
-keys for experimental use or publicly known keys.
+Do not use regular or sensitive viewing keys with Zebra's experimental scanning feature. Do not use this feature on a shared machine. We suggest generating new keys for experimental use or publicly known keys.
 
 ## Build & Install
 
-Use [Zebra 1.6.0](https://github.com/ZcashFoundation/zebra/releases/tag/v1.6.0)
-or greater, or the `main` branch to get the latest features, and enable the
-`shielded-scan` feature during the build. You can also use Rust's `cargo` to
-install the latest release:
+Use [Zebra 1.9.0](https://github.com/ZcashFoundation/zebra/releases/tag/v1.9.0) or greater, or the `main` branch to get the latest features.
+
+You can also use Rust's `cargo` to install `zebra-scanner` from the latest release Zebra repository:
 
 ```bash
-cargo install --features shielded-scan --locked --git https://github.com/ZcashFoundation/zebra zebrad
+cargo install --locked --git https://github.com/ZcashFoundation/zebra zebra-scan
 ```
 
-Zebra binary will be at `~/.cargo/bin/zebrad`, which should be in your `PATH`.
+The scanner binary will be at `~/.cargo/bin/zebra-scanner`, which should be in your `PATH`.
 
-## Configuration
+## Arguments
 
-Generate a configuration file with the default settings:
+Retrieve the binary arguments with:
 
 ```bash
-zebrad generate -o ~/.config/zebrad.toml
+zebra-scanner --help
 ```
 
+## Scanning the Block Chain
 
-In the generated `zebrad.toml` file, use:
-
-- the `[shielded_scan]` table for database settings, and
-- the `[shielded_scan.sapling_keys_to_scan]` table for diversifiable full viewing keys.
-
-Sapling diversifiable/extended full viewing keys strings start with `zxviews` as
-described in
-[ZIP-32](https://zips.z.cash/zip-0032#sapling-extended-full-viewing-keys).
-
-For example, to scan the block chain with the [public ZECpages viewing
-key](https://zecpages.com/boardinfo), use:
+Before starting, ensure a `zebrad` node is running locally with the RPC endpoint open. Refer to the [lightwalletd zebrad setup](https://zebra.zfnd.org/user/lightwalletd.html#configure-zebra-for-lightwalletd) or [zebrad mining setup](https://zebra.zfnd.org/user/mining.html#configure-zebra-for-mining) for instructions.
```

Comment:
Optional: We could be more specific about what's needed here too, but "is running locally" also seems fine, hopefully this won't be a constraint in the future anyway.